infra/core/managed-cluster/aks-cluster.bicep

```diff
@@ -60,6 +60,36 @@ var asoPlatformTeamMiRbacs = [
   }
 ]
 
+var systemNodePool = {
+  name: 'npsystem01'
+  mode: 'System'
+  count: cluster.npSystem.count
+  vmSize: 'Standard_DS2_v2'
```

Comment:
DS3_V2 - but also needs to be variables that can change on environments.

Reply:
2 diff vars for 2 diff nodepools, user, system, blah, should match naming conventions